Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM busybox
+
+ENV SHARED_PATH /etc/mnt/shared-pv
```

Comment:
ENV - define envs during runtime
ARG - define envs during build

COPY command runs during build

in this case you did not define ARG SHARED_PATH so it will be "" always during build